### PR TITLE
Meta fields

### PR DESCRIPTION
**Pull request by jmbarne3**
Creating the alias and color meta fields for the program_types custom post type.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -1,6 +1,44 @@
...
+};
+
+gulp.task('jshint', function() {
+  gulp.src(config.jsPath + '/*.js')
+    .pipe(jshint())
+    .pipe(jshint.reporter('fail'));
+});
+
+gulp.task('js-frontend', ['jshint'], function() {
+  var minified = [
```
Comment (cjg89) [inline gulpfile.js]: Should this be empty?

Comment (jmbarne3) [inline gulpfile.js]: I set this up in anticipation of some front end javascript needing to be written for a degree-search style typeahead.

Comment (cjg89) [inline gulpfile.js]: Ok, cool 👍

**Comment by keeeeeegan** [comment]
Looks good

**Pull request merged**